### DISCUSSION TITLE
What additional process types are needed?

### DISCUSSION THREAD

**Discussion by abmoore03** [abmoore03/scubed_community#3] [General]
sCubed includes a controlled list of process types. You can review the controlled list via the sCubed add-on: Select `Explore controlled terms` and then select one of the buttons called `Process type`.

What types of processes are not present in the list and would benefit you?

It's helpful to relay the name of the process, an ontology ID (if available),  and any attributes of the process, e.g.

- **Process type**: centrifugation,
- **Ontology term**: MSIO_0000146, and
- **Attributes**:  temperature, speed.

Not sure how to find ontology terms? You can search for ontology terms on [NCBI BioPortal](https://bioportal.bioontology.org/) or [EBI OLS](https://www.ebi.ac.uk/ols4).  If you're not sure what the ID for a term is, you can just give us the link.

**Comment by esselmanc123** [comment]
Hello. I have added some process types below:

1. 
     Process type: vacuum centrifuge concentrator
     Ontology: https://bioportal.bioontology.org/ontologies/ERO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FERO_0000244
     Attributes: Temperature, Time, speed
     
2. 
     Process type: homogenization by hand
     Ontology: https://bioportal.bioontology.org/ontologies/NPO?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2Fnpo%23NPO_1959
  
3. 
     Process type: vortex
     Ontology: http://purl.obolibrary.org/obo/NCIT_C29544
     Attributes:Speed
     
4. 
    Unsure about solvent extraction. Not in the controlled terms tab. 
    
Thanks,
Chris
